CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- Bump `stm32f1` dependency (`0.10.0`)
```

Comment:
It's a breaking change, right?

Reply:
I don't remember any change for `f1` in `stm32-rs`.

Reply:
Technically it is as this is a semver incompatible version export. If a user depend on stm32f1 0.9 and this half, it will break compilation as the 2 types will be different.